docs/project/dogfooding.md

```diff
@@ -6,6 +6,23 @@ This document provides the steps necessary to consume a nightly build of
 Please note that these steps are likely to change as we're simplifying
 this experience. Make sure to consult this document often.
 
+## Obtaining nightly builds of NuGet packages
+
+If you are only looking to get fixes for an individual NuGet package, and don't need a preview version of the entire runtime, you can add the following package feed to `NuGet.config`:
+
+**MSFT TODO:** Someone at Microsoft should confirm if any other feeds might be needed here
+
+```xml
```

Comment:
The easiest way to do this is by running:

`dotnet nuget add source -n dotnet5 https://dnceng.pkgs.visualstudio.com/public/_packaging/dotnet5/nuget/v3/index.json`

Maybe just mention this command here, and refer to docs for more details?

Reply:
I often create a solution-scoped NuGet.Config in these situations as opposed to using the user-scoped one, so I was somewhat trying to avoid a command with specific behavior and just focusing on giving the necessary info.  However, I think I agree that this command with a link to the docs you mentioned below is probably worthwhile.

Reply:
I agree that this is a good idea to create a local NuGet.config in these cases. Maybe recommend doing this here, so that these instructions do the right thing for people who just want to quickly copy&paste something ?

```
dotnet new nuget
dotnet nuget add ...
```

Reply:
I'm not so much of a command line guy ... I know, feel free to boo me .... but is there any difference between `dotnet new nuget` and `dotnet new nugetconfig`?  It seems like they are probably identical.  Is one preferred over the other?

Reply:
`dotnet new nugetconfig` is the proper command.

The CLI allows the names to be shortened, so `dotnet new nuget` works fine as long as there is no ambiguity.